indico_toolkit/pipelines/file_processing.py

```diff
@@ -45,6 +45,16 @@ def get_file_paths_from_dir(
             f"Found {len(self.file_paths)} valid files and {len(self.invalid_suffix_paths)} paths with invalid suffixes."
         )
 
+    def move_all_file_paths(
+        self, 
+        path_to_dir,
```

Comment:
`path_to_dir` is a bit confusing. It should be something more clear like ``input_dir`